test/multiple-steps/build-with-more-steps.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2018 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: build.knative.dev/v1alpha1
+kind: Build
+metadata:
+  name: test-build-with-more-steps
```

Comment:
We have a few tests that happen to specify multiple steps, e.g., `test-custom-volume` and `test-home-volume`.

It's a real limitation of our integration testing "framework" that the only way we have right now to test things against a real cluster is to send builds and expect them to succeed or fail. Once we have a more "real" framework in place (stolen from `knative/serving` no doubt), we can have a test that runs a multi-step build and watches to see multiple status updates.

Until then, I'm not sure this test really adds much, except that it's proof that a build that has two steps succeeds.

Reply:
I completely agree with you on this.